pyproject.toml

```diff
@@ -1,9 +1,11 @@
 [tool.black]
 line-length = 100
 
-# changes made here should also be made in `.pre-commit-config.yaml` and `Makefile`
 [tool.ruff]
 line-length = 100
+
+# -- changes made here should also be made in `.pre-commit-config.yaml` and `Makefile` --
+[tool.ruff.lint]
```

Comment:
This gets rid of the warning from `ruff`. They recently segregated linting options from those that reformat the code.